### DISCUSSION TITLE
Param "auth_attempts" > additional explanation.

### DISCUSSION THREAD

**Discussion by Yuri-01** [AdguardTeam/AdGuardHome#3149] [Q&A]
Hi.

Please explan, what time period for this failed login attempts?

**Comment by ameshkov** [comment]
Depends on `block_auth_min` (minutes to block)

See https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration

**Reply by Yuri-01** [reply]
Здравствуйте.
Видимо, я не достаточно точно выразил свою мысль...

Указанный вами параметр (block_auth_min) определяет продолжительность блокировки.
Это не то, о чём я спрашивал: я хотел узнать, каков тот период времени в продолжении которого счётчик неудачных  попыток авторизации увеличивается (5 мин., 1 час, ...).

Дополнение:
Часто при вводе в действие password policy специфицирую 3-и параметра:
1.число неудачных попыток авторизации;
2.период  времени, в течении которого эти неудачные попытки были совершены;
3.время, на которое блокируется уч.запись пользователя.
Я вижу, что у вас отделены 1-ый и 3-ий параметры. А каково значение 2-го?

**Reply by ameshkov** [reply]
Тогда @EugeneOne1 ответит, я, честно говоря, не помню.

**Reply by Yuri-01** [reply]
Спасибо.

**Reply by EugeneOne1** [reply]
@Yuri-01, если я вас правильно понял, то вот значение, которое вас интересует (1 минута):
https://github.com/AdguardTeam/AdGuardHome/blob/a031cae447e5f77a88f9d7a8c2c1d1dcb1fda11c/internal/home/authratelimiter.go#L10

**Reply by Yuri-01** [reply]
Понял вас. Спасибо.
